lib/dbPool.js

```diff
@@ -190,6 +193,21 @@ class DBPool {
     }
   }
 
+  /**
+   * @description
+   * Configuration option to return numeric types as a Number instead of a String.
+   * When set to true numeric types for `DBPool.runSql` and `DBPool.prepareExecute` 
+   * will be returned as a Number instead of a String.
+   * @param {boolean} flag - option to return numeric types as a Number instead of a String
+   * @returns {boolean} - resolves to true/false indicating the state of the flag
+   */
+  enableNumericTypeConversion(flag) {
+    if(typeof flag !== 'undefined') {
```

Comment:
We should probably just check if the passed flag is a boolean and set the `enableNumericTypeConversion` flag
```suggestion
    if(typeof flag !== 'boolean') {
      this.enableNumericTypeConversion = flag;
    }
    return this.enableNumericTypeConversion;
  }
```